<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>anaconda on 111qqz的小窝</title><link>https://111qqz.github.io/tags/anaconda/</link><description>Recent content in anaconda on 111qqz的小窝</description><generator>Hugo -- gohugo.io</generator><language>zh-cn</language><lastBuildDate>Thu, 08 Jun 2017 13:03:51 +0000</lastBuildDate><atom:link href="https://111qqz.github.io/tags/anaconda/index.xml" rel="self" type="application/rss+xml"/><item><title>conda升级anaconda　ValueError的解决办法</title><link>https://111qqz.github.io/2017/06/how-to-fix-conda-upgrade-valueerror/</link><pubDate>Thu, 08 Jun 2017 13:03:51 +0000</pubDate><guid>https://111qqz.github.io/2017/06/how-to-fix-conda-upgrade-valueerror/</guid><description>conda update anaconda 后提示 ValueError: unsupported format character ')' (0x29) at index 49 查到了这个：anaconda update issue I have narrowed this down to the following packages: package build psutil-1.2.1 py27_0 hard-link pycparser-2.10 py27_0 hard-link pykit-0.1.0 np18py27_2 hard-link pyparsing-2.0.1 py27_0 hard-link by calling "conda install anaconda" and then successfully installing everything else one at a time. These</description></item></channel></rss>